int32 x
int32 y
float32 angle
int32 class_id
int32 octave
float32 response
float32 size
